CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ add_custom_target(clean_locks)
 add_custom_target(all_xc7_tests)
 add_custom_target(all_ice40_tests)
 add_custom_target(all_quicklogic_tests)
+add_dependencies(all_quicklogic_tests
+    all_ql_tests
+    )
```

Comment:
This dependency should be added in the quicklogic branch

Reply:
under `quicklogic/pp3/tests/CMakeFiles.txt` after defining the `all_ql_tests` target

Reply:
Moved dependency to `quicklogic/pp3/tests/CMakeFiles.txt`